Feature Text Rephrase Page

2012-Pai-Raymond opened the pull request:
Added in a text rephrase page. Currently, it only reverse string in backend to see if the connection is working. Notable difference between this page and sentiment analysis is that the output field is a textarea that will display the result. Also, I've removed textarea resizing on both pages.

2012-Pai-Raymond committed:
Added Text Rephrase Page

2012-Pai-Raymond committed:
Fixed a bug in Sentiment Analysis, where consecutive input overflows output field

2012-Pai-Raymond committed:
Added Sentiment Analysis Unit Test to see function has been called

2012-Pai-Raymond committed:
Written more unit test for sentiment analysis

2012-Pai-Raymond committed:
Mispelled 'Rephrase'

2012-Pai-Raymond committed:
Added more component testing for Sent Analysis

agentz101 commented:
Looks good to me.

BettridgeKameron commented:
Looks mostly good, but a few things:
1. is there a reason for NLTK to have quiet=False?
2. Can you center the rephrase button? (check image below)
3. __snapshot__ folder shouldn't committed, can you remove it from the commit, and add it to .gitignore?
4. can you fix the text alignment inside the box? notice how this looks a bit weird where the left side and right side are different (check image below)
5. can the output box color match the one in sentiment analysis? Feel free to copy code from that (we could even make a OutputBox component in the future).
![image](https://github.com/BettridgeKameron/LASS/assets/88714889/9e6d28c8-3445-41c4-af01-f34eb27d4a34)

2012-Pai-Raymond committed:
__snapshots__

2012-Pai-Raymond committed:
Fixed all alignment and implement color for output

2012-Pai-Raymond committed:
reformatted code

2012-Pai-Raymond committed:
center justified error message

2012-Pai-Raymond commented:
Fixed with new changes

> Looks mostly good, but a few things:
> 
> 1. is there a reason for NLTK to have quiet=False?
> 2. Can you center the rephrase button? (check image below)
> 3. **snapshot** folder shouldn't committed, can you remove it from the commit, and add it to .gitignore?
> 4. can you fix the text alignment inside the box? notice how this looks a bit weird where the left side and right side are different (check image below)
> 5. can the output box color match the one in sentiment analysis? Feel free to copy code from that (we could even make a OutputBox component in the future).
>    ![image](https://private-user-images.githubusercontent.com/88714889/309913450-9e6d28c8-3445-41c4-af01-f34eb27d4a34.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk1OTYwODMsIm5iZiI6MTcwOTU5NTc4MywicGF0aCI6Ii84ODcxNDg4OS8zMDk5MTM0NTAtOWU2ZDI4YzgtMzQ0NS00MWM0LWFmMDEtZjM0ZWIyN2Q0YTM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA0VDIzNDMwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1OTZhY2I4YjU0OGIwN2RiZTMxMjZjYTE1NDA3YzhjZDRhOGE2YjhiNWZiZTMzYTQ1YmMxYTdiYzRiZmVjNTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cSutiqwB87HrRFNXjtTYsQ27-GkPMeJPx9eBrQ8UJi0)

Fixed with the new changes.
